### PR TITLE
GA Migration Python PreCommit Sharding Runners Portability Flink

### PR DESCRIPTION
**Pull request by MarcoRob**
As part of the migration of Precommit and Postcommit Jobs from Jenkins to GA in self-hosted runners, this PR contains:

- Migrated workflow [job-precommit-python-runners-portability-flink.yml](https://github.com/apache/beam/blob/4d2bba1d60e56386e2c7fe57406398ff0ea4c9b4/.github/workflows/job-precommit-python-runners-portability-flink.yml)

The migrated workflow was added to [CI.md](https://github.com/apache/beam/blob/4d2bba1d60e56386e2c7fe57406398ff0ea4c9b4/CI.md)

`DO NOT MERGE` until the effort to use self-hosted runners is completed https://github.com/apache/beam/pull/22703

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: .github/workflows/job-precommit-python-runners-portability-flink.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+
+name: Python Precommit Runners Portability Flink
+on:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+    paths: [ 'sdks/python/apache_beam/runners/portability/flink_runner.py',
```
Comment (damccorm) [inline .github/workflows/job-precommit-python-runners-portability-flink.yml:28]: We should run postcommits on all paths, please remove these

**Review by damccorm — COMMENTED** [review]
Looks good other than one comment

File: .github/workflows/job-precommit-python-runners-portability-flink.yml (hunk 1 above)
Comment (MarcoRob) [reply .github/workflows/job-precommit-python-runners-portability-flink.yml:28]: Same [context](https://github.com/apache/beam/pull/23364#discussion_r999969683) @damccorm

Comment (MarcoRob) [reply .github/workflows/job-precommit-python-runners-portability-flink.yml:28]: is there any comment?

**Pull request closed without merging**